packages/slider/stories/slider.stories.ts

```diff
@@ -132,6 +132,81 @@ export const Default = (args: StoryArgs = {}): TemplateResult => {
     `;
 };
 
+export const Filled = (args: StoryArgs = {}): TemplateResult => {
+    return html`
+        <div style="width: 500px; margin-inline: 20px;">
+            <sp-slider
+                max="1"
+                variant="filled"
+                min="0"
+                value=".5"
```

Comment:
Nit, can we use a value other than the middle of the range so that this story shows the fill by default in VRT so that we can test that the output stays the same over time.

Reply:
<img width="782" alt="Screenshot 2024-01-17 at 11 34 00 AM" src="https://github.com/adobe/spectrum-web-components/assets/8790510/dbfba5ee-72bb-4520-a27c-8d451f9faf76">

Reply:
I'm sorry, I put this comment on the wrong story. 😞 I meant for https://bug-slider-filled--spectrum-web-components.netlify.app/storybook/?path=/story/slider--fill-start for all the same reasons. I swear this is the last thing!

Reply:
<img width="788" alt="Screenshot 2024-01-17 at 6 45 46 PM" src="https://github.com/adobe/spectrum-web-components/assets/8790510/ff2dcf3e-3c49-45b5-a049-479905366299">